Makefile.main

```diff
@@ -46,17 +46,13 @@ DEV_PREFIX := dev
 VERSION ?= $(DEV_PREFIX)-$(COMMIT)$(GIT_DIRTY)
 
 # Travis CI
-ifeq ($(TRAVIS), true)
-	VERSION := $(TRAVIS_BRANCH)
+ifneq ($(TRAVIS_TAG), )
+	VERSION := $(TRAVIS_TAG)
 endif
 
 # Drone CI
-ifeq ($(DRONE), true)
-	ifeq ($(DRONE_BUILD_EVENT), tag)
-		VERSION := $(DRONE_TAG)
-	else
-		VERSION := $(DRONE_BRANCH)
```

Comment:
Just wondering, if this change somehow will not affect https://github.com/src-d/guide/blob/master/engineering/continuous-delivery.md#branches

Reply:
:thinking: I don't see any obvious problem with that, the build version name should not interfere in any way. Do you see any potential problems?